build/conda/cleanup_artifacts.bat

```diff
@@ -0,0 +1,44 @@
+@echo off
```

Comment:
@andrewkenreich this batch script was automatically translated from the shell script that cleans up the conda artifacts that make hybrid (conda-poetry) enviromnents buggy. As a windows guy, can you help me validate that it works safely please?

Reply:
thanks @andrewkenreich